submissions/Neskazhuk/popup/styles.css

```diff
@@ -0,0 +1,167 @@
+/* General */
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+  font-weight: 600;
+  color: gray;
+}
+
+.link {
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav-list,
+.popup-list {
+  list-style: none;
+}
+
+/* Nav */
+.nav {
+  height: 50px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 20px;
+  margin-right: 40px;
+  margin-top: 25px;
+}
+
+.nav-item {
+  padding:5px;
+  cursor: pointer;
+  border:2px solid transparent;
+  border-radius: 10px;
+}
+
+.nav-item:hover {
+  border-color: #7b68ee;
+}
+
+/* Icons */
+.popup-icon,
+.avatar {
+  display: block;
+  width: 1.6em;
+  height: 1.6em;
+  cursor: pointer;
+}
+
+/* Checkboxes */
+.popup-input,
+.more-input {
+  position: absolute;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.popup-input:focus-visible ~ label > img {
```

Comment:
Avoid styling by HTML tags. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, [read about CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
changed some tag styling, using classes instead